packages/devtools_shared/test/server/devtools_extensions_api_test.dart

```diff
@@ -146,14 +146,14 @@ void main() {
 
   group(ExtensionsApi.apiExtensionEnabledState, () {
     late File optionsFile;
-    late final optionsFileUriString = p.join(
+    late final optionsFileUriString = p.posix.join(
       extensionTestManager.runtimeAppRoot,
       devtoolsOptionsFileName,
     );
 
     setUp(() async {
       await initializeTestDirectory();
-      optionsFile = File.fromUri(Uri.file(optionsFileUriString));
+      optionsFile = File.fromUri(Uri.parse(optionsFileUriString));
```

Comment:
`Uri.file()` takes a file path, not a URI. It worked for non-Windows but explodes on Windows. `Uri.parse()` takes a URI in a string which is what `optionsFileUriString` is.